src/marketlist.schema.json

```diff
@@ -42,7 +42,7 @@
       "minLength": 1,
       "maxLength": 30,
       "pattern": "^[\\w ]+$",
-      "examples": ["yearn", "stablecoin", "money market", "protection market"]
+      "examples": ["yearn", "stablecoin", "money market", "market"]
```

Comment:
I think we can keep "protection market" as an example given that "money market" is also an example